### ISSUE TITLE
Stumped on how to debug app

### ISSUE THREAD

**Issue by maxwellE** [bazel-contrib/publish-to-bcr#174]
Hello All,

Thank you so much for making this very important tool for the Bazel community. I would love to use this tool but I cannot seem to figure out where I am going wrong with my package https://github.com/Tinder/bazel-diff . I have triple checked my config in https://github.com/Tinder/bazel-diff/tree/master/.bcr and made sure my permissions are all lined up on each repo. What action can I take next to figure out what is wrong? Thanks

**Comment by kormide** [comment]
Hi @maxwellE. Looking at the logs, the last publish attempt appears to have been on Oct. 7 and that failed because the app wasn't installed to a fork of the Bazel Central Registry. Can you verify that the app is installed both to your ruleset as well as BCR fork either in your `maxwellE` account or in the `Tinder` org?

**Comment by maxwellE** [comment]
I just confirmed the app is properly set up on 

https://github.com/Tinder/bazel-diff

and

https://github.com/maxwellE/bazel-central-registry

I just made the release, hoping it works https://github.com/Tinder/bazel-diff/releases/tag/8.1.2

**Comment by kormide** [comment]
Hm, there was nothing in the logs. Do you have the correct app installed?

[This](https://github.com/apps/publish-to-bcr) is the correct one. [This](https://github.com/apps/publish-to-bcr-bot) is not the correct one.

**Comment by maxwellE** [comment]
As long as I publish a release and make a tag it should work right?

**Comment by kormide** [comment]
> As long as I publish a release and make a tag it should work right?

Yeah that's right. And if it failed then I should at least see some logs.. I notice that you set the fixedReleaser to `maxwellE` but the release is published by `tinder-maxwellelliott`. Can you try changing the fixedReleaser and installing the app to a fork of the BCR under your other account?

**Comment by kormide** [comment]
> Okay I tried on tinder-maxwellelliott, do you see any logs?

There are no log entries for your release. I'll make some time to dive deeper on this and I'll get back to you.

**Comment by kormide** [comment]
@maxwellE how does `release.tar.gz` get uploaded to your releases? Is this done manually or is it automated somewhere? I suspect that when Publish to BCR kicks off on a release it can't find the archive to download. The app uses backoff and retries, so if the archive is uploaded shortly after the release is created then it should still work.

**Comment by maxwellE** [comment]
I do it manually but its part of the release when I submit it

**Comment by kormide** [comment]
> I do it manually but its part of the release when I submit it

Do you have a script or anything you can share with me so I can try to replicate it on a fork of your repo in my dev environment? You can also reach me on the Bazel slack if that's easier.
